src/ess/reduce/nexus/generic_workflow.py

```diff
@@ -220,24 +220,34 @@ def assemble_monitor_data(
     assemble_detector_data,
 )
 
+_default_params = {
+    PulseSelection: PulseSelection(()),
```

Comment:
Why is this needed now? It seems unrelated to the other changes.

Reply:
Avoids 3x code duplication?

Reply:
You are right, that was a bad rebase, fixing.